checks/evaluation/vulnerabilities.go

```diff
@@ -16,45 +16,41 @@ package evaluation
 
 import (
 	"fmt"
-	"strings"
-
-	"github.com/google/osv-scanner/pkg/grouper"
 
 	"github.com/ossf/scorecard/v4/checker"
 	sce "github.com/ossf/scorecard/v4/errors"
+	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes/hasOSVVulnerabilities"
 )
 
 // Vulnerabilities applies the score policy for the Vulnerabilities check.
-func Vulnerabilities(name string, dl checker.DetailLogger,
-	r *checker.VulnerabilitiesData,
+func Vulnerabilities(name string,
+	findings []finding.Finding,
+	dl checker.DetailLogger,
 ) checker.CheckResult {
-	if r == nil {
-		e := sce.WithMessage(sce.ErrScorecardInternal, "empty raw data")
+	expectedProbes := []string{
+		hasOSVVulnerabilities.Probe,
+	}
+
+	if !finding.UniqueProbesEqual(findings, expectedProbes) {
+		e := sce.WithMessage(sce.ErrScorecardInternal, "invalid probe results")
 		return checker.CreateRuntimeErrorResult(name, e)
 	}
 
-	aliasVulnerabilities := []grouper.IDAliases{}
-	for _, vuln := range r.Vulnerabilities {
-		aliasVulnerabilities = append(aliasVulnerabilities, grouper.IDAliases(vuln))
+	vulnsFound := negativeFindings(findings)
+	numVulnsFound := len(negativeFindings(findings))
+	for _, vuln := range vulnsFound {
+		dl.Warn(&checker.LogMessage{
+			Text: vuln.Message,
+		})
```

Comment:
What's the difference between this and just `checker.LogFindings`?

Reply:
Nothing. Switching.